app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class Post < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] The post should belong to an `:author`, not `:user`. You can use the `class_name` option to specify which model you are referring to. You should have something like this.

  ```
  belongs_to :author, class_name: 'User'
  ```

- [ ] From your database schema I can see you are referencing the `user` in the posts table. You need to run new migrations and reference the `author` to match this requirement. You can point to the user table using the `to_table` option.

  You can look at this thread: [https://stackoverflow.com/questions/13694654/specifying-column-name-in-a-references-migration](https://stackoverflow.com/questions/13694654/specifying-column-name-in-a-references-migration)